gnuplot.yaml

```diff
@@ -0,0 +1,7 @@
+name: gnuplot
+sources:
+  - type: archive
+    url: https://downloads.sourceforge.net/gnuplot/gnuplot-5.2.5.tar.gz
+    sha256: 039db2cce62ddcfd31a6696fe576f4224b3bc3f919e66191dfe2cdb058475caa
+post-install:
+  - install -p -D -m 0644 "Copyright" -t "${FLATPAK_DEST}/share/licenses/gnuplot/";
```

Comment:
The files isn't exported, it's part of Sources extension.

Reply:
Done.

https://github.com/flathub/flathub/pull/801/commits/de93f9e911a070beb1913aade48a4b7f544a2eb9#diff-2df43c7cf78c1e824a1ba0b19a147a95L7